src/ralph/sim_cards/admin.py

```diff
@@ -43,6 +45,16 @@ class SIMCardAdmin(RalphAdmin):
         }),
     )
 
+    def get_multiadd_fields(self, obj=None):
+        multi_add_fields = [
+            {'field': 'card_number', 'allow_duplicates': False},
+            {'field': 'phone_number', 'allow_duplicates': False},
+            {'field': 'pin1', 'allow_duplicates': False},
```

Comment:
allow_duplicates = True - for pin and for puk

Reply:
done